ts/store/reducers/__tests__/backendStatus.test.ts

```diff
@@ -6,13 +6,17 @@ import {
   areSystemsDeadReducer,
   BackendStatusState,
   bpdRankingEnabledSelector,
+  isPremiumMessagesOptInOutEnabledSelector,
   isUaDonationsEnabledSelector,
   sectionStatusSelector,
   uaDonationsBannerConfigSelector
 } from "../backendStatus";
 import { GlobalState } from "../types";
 
-jest.mock("../../../config", () => ({ uaDonationsEnabled: true }));
+jest.mock("../../../config", () => ({
+  uaDonationsEnabled: true,
+  premiumMessagesOptInEnabled: true
+}));
```

Comment:
Instead of mocking it locally on this test, what about doing it globally [here](https://github.com/pagopa/io-app/blob/master/ts/__mocks__/react-native-config.ts)?